nucypher/utilities/prometheus/metrics.py

```diff
@@ -214,16 +218,17 @@ def create_staking_events_metric_collectors(ursula: 'Ursula', metrics_prefix: st
     collectors: List[MetricsCollector] = []
     staking_agent = ContractAgency.get_agent(StakingEscrowAgent, registry=ursula.registry)
 
+    staker_address = ursula.checksum_address
```

Comment:
This is subject to change, and will likely cause this collector to break when it does.